src/views/tenants/tenants.jsx

```diff
@@ -1,140 +1,273 @@
+          setCheckboxRenderCount(checkboxRenderCount + 1);
+        })
+        .then(() => setSelectedTenants([]))
+        .catch((error) => {
+          Toast(error.message, "error");
+          console.log(error)
+        })
+    ))
+    fetchAllTenants();
+    toggleArchiveModal();
+  }
+
+  return (
+    <div className='main-container'>
+      <div className='tenants'>
+        <div className='section-header'>
+          <h2 className='page-title'>Tenants</h2>
+          <Link
+            className='button is-primary is-rounded ml-4'
+            to='/add/tenant'>+ ADD NEW</Link>
+        </div>
+        <div className='search-and-archive-container'>
+          <Search
+            input={allTenants}
+            outputLocation={searchedTenants}
+            isFilteredLocation={isSearchActive}
+            setIsFilteredStateFalse={handleDisableSearch}
+            setOutputState={handleSearchOutput}
+            placeholderMessage="Search tenants by name, property, or JOIN staff" />
+          <ShowHideSwitch
+            id="housedToggleSwitch"
+            labelText="Unhoused:"
+            isShowState={showHoused}
+            handleToggleChange={handleToggleHoused}
+          />
+
+          <ShowHideSwitch
+            id="archivedToggleSwitch"
+            labelText="Archived:"
+            isShowState={showArchived}
+            handleToggleChange={handleToggleArchived}
+          />
+
+        </div>
+        <div className='bulk-actions-container py-3'>
+          <button
+            className={`button is-rounded is-primary ml-3 ${selectedTenants.length && 'is-active-button'}`}
+            onClick={toggleArchiveModal}
+          >
+            <FontAwesomeIcon
+              className="mr-3"
+              icon={faArchive}
+            />
+            Archive Tenants
+          </button>
+        </div>
+
+        <div className="tenants-list">
+          <BootstrapTable
+            key={`tables-of-tenants--${checkboxRenderCount}`} wrapperClasses='tenants-list-wrapper'
+            keyField='id'
+            data={displayTenants}
+            columns={columns}
+            selectRow={({
+              mode: 'checkbox',
+              clickToSelect: true,
+              onSelect: (row, isSelect) => isSelect ? handleSelectRow(row) : handleDeselectRow(row),
+              onSelectAll: (isSelect, rows) => isSelect ? handleSelectAll(rows) : handleDeselectAll(rows),
+              sort: true,
+              headerColumnStyle: () => ({ width: "5%" }),
+              nonSelectable: nonSelectableRows,
+              nonSelectableStyle: () => ({ color: '#999999' })
+            })}
+            defaultSorted={[
+              {
+                dataField: 'propertyName',
+                order: 'asc'
+              }]}
+            bootstrap4={true}
+            headerClasses='table-header'
+          />
+        </div>
+
+
+      </div >
+      {showArchiveModal &&
+        <Modal
+          content={
```

Comment:
Let's add the titleText attribute to the modal so there's a header to it like the other modals